device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/IotHubTransport.java

```diff
@@ -1238,7 +1238,7 @@ private void reconnect(TransportException transportException)
             reconnectionAttempts++;
 
             RetryPolicy retryPolicy;
-            if (this.deviceClientConfigs.size() != 1)
```

Comment:
Assuming that configs.size != 1 means multiplexing is an incorrect assumption, so that is corrected here. The ```isMultiplexing``` flag is set at constructor time and is determined higher up in the code. For instance, when a IotHubTransport instance is created by a multiplexingClient, this flag is set to true.